### PR TITLE
Blog page

### PR DESCRIPTION
**Pull request by lucacasonato**
A place for future release posts and announcements

### CODE REVIEW

File: components/Footer.tsx [hunk 1]
```diff
@@ -53,9 +53,9 @@ const Footer = (props: { simple?: boolean }) => (
           </Link>
         </div>
         <div className="px-5 py-2">
-          <Link href="/v1">
+          <Link href="/blog">
             <a className="text-base leading-6 text-gray-500 hover:text-gray-900">
-              v1.0 Announcement
+              Blog
```
Comment (ry) [inline components/Footer.tsx:58]: News? Updates?

Comment (lucacasonato) [reply components/Footer.tsx:58]: News sounds good

File: pages/posts/index.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+              const date = new Date(post.publish_date);
+              const format = new Intl.DateTimeFormat(undefined, {
+                month: "long",
+                day: "numeric",
+                year: "numeric",
+              });
+              return (
+                <div key={post.id}>
+                  <p className="text-sm leading-5 text-gray-500">
+                    <time dateTime="2020-03-16">{format.format(date)}</time>
```
Comment (kidonng) [inline pages/posts/index.tsx:61]: Why is the `dateTime` attribute's value fixed? And also it should be `datetime`.

File: pages/posts/index.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                  </p>
+                  <Link href="/posts/[article]" as={`/posts/${post.id}`}>
```
Comment (kidonng) [inline pages/posts/index.tsx:63]: Possibly should be `[post]` instead of `[article]`

**Pull request merged**